chips/sifive/src/plic.rs

```diff
@@ -46,10 +46,16 @@ impl Plic {
         }
     }
 
-    /// Clear all pending interrupts.
+    /// Clear all pending interrupts. Calims each interrupt.
```

Comment:
Please detail here why interrupts need to be claimed first and add a link to the manual.

Reply:
Done.